src/main/java/org/zalando/baigan/repository/S3ConfigurationRepository.java

```diff
@@ -65,15 +67,15 @@ public void put(@Nonnull String key, @Nonnull String value) {
     }
 
     private void loadConfigurations() {
-        LOG.info("Loading configurations from S3 file");
+        LOG.debug("Loading configurations from S3 file {}", configFileS3Path);
```

Comment:
What about simply something a la...

```
Loading configuration from bucket [{bucket}] at [{key}]...
```

... and the other log statements accordingly. As easy for the human to understand, but without the "cost" (in the sense of code comprehension) of the additional field.

Reply:
Ah wait these fields are both "hidden" in `S3FileLoader` :bulb:

Reply:
maybe `S3FileLoader#getS3Path` ?

Reply:
The bucket and key variables are not available to access either. They are only constructor parameters. So it would be two additional fields (although easier to reason about as the fields exist in the constructor in the same way). I'm impartial here. I will change it as you suggested (tomorrow ;) ).

Reply:
In `S3FileLoader` they are already fields, just not accessible yet. I feel with making them accessible via a (computing) `getS3Path` we kinda hit best of both worlds.